backend/dataline/services/llm_flow/graph.py

```diff
@@ -41,7 +45,11 @@ def __init__(
         self,
         dsn: str,
     ) -> None:
-        self.db = SQLDatabase.from_uri(dsn)
+        # Enable this try catch once we support errors with streaming responses
+        try:
+            self.db = SQLDatabase.from_uri(dsn)
+        except Exception as e:
+            forward_connection_errors(e)
```

Comment:
@RamiAwar `forward_connection_errors` doesn't always raise an exception. The code below might still run if no exception is raised, and we wouldn't want that. Example: `self.db` would be None, and two lines below we're using it